specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -0,0 +1,209 @@
+
+The client receives the Self-Issued ID Token (containing an access token based on the `bearer_access_scope` value) and
+provides it with a request to access protected resources to the verifier. The verifier extracts the access token and
+uses it to request VPs from the client's CS. The verifier may resolve the CS endpoint through a variety of methods,
+for example, by resolving the client's DID document and using a service entry as described
+in [Section](#6-cs-endpoint-resolution-through-did-documents). The VPs are then returned to the Verifier.
+
+# 3. Security
+
+CS endpoints may require an [OAuth 2](https://www.rfc-editor.org/rfc/rfc6749) access token obtained from the resource
+owner. For example, a client (the resource owner) that needs to present a VP to an endpoint will provide an access token
+to the endpoint. The endpoint server will in turn use the access token when resolving the VP through a request to the
+client's Credential Service.
+
+Tokens may be [OAuth 2.0 Bearer Tokens](https://www.rfc-editor.org/rfc/rfc6750) or any other valid OAuth 2.0 token type.
+Any OAuth 2.0 grant type may be used to request the access tokens.
+
+## 3.1. Access Scopes
+
+Scopes are used to specify access privileges. A scope is a string value in the form:
+
+`[alias]:[descrimiator]:[read|write|all]`
+
+The `[alias]` value may be implementation-specific. The `all` value indicates both read and write access.
+
+### 3.1.1. The `org.eclipse.tractusx.vc.type` Alias
+
+The `vc.type` alias value must be supported and is used to specify access to a verifiable credential by type. For
+example:
+
+`org.eclipse.tractusx.vc.type:Member:read`
+
+denotes read-only access to the VC type `Member` and may be used to request a VC or VP.
+
+### 3.1.2. The `org.eclipse.tractusx.vc.id` Alias
+
+The `org.eclipse.tractusx.vc.id` alias value must be supported and is used to specify access to a verifiable credential
+by id. For
+example:
+
+`org.eclipse.tractusx.vc.id:8247b87d-8d72-47e1-8128-9ce47e3d829d:read`
+
+denotes read-only access to the VC identified by `8247b87d-8d72-47e1-8128-9ce47e3d829d` and may be used to request a VC
+or VP.
+
+### 3.1.3. The `*` Wildcard Alias
+
+Implementations must also support the `*` wildcard:
+
+`org.eclipse.tractusx.vc.type:*:write`
+
+The above expression enables write-only access to all VCs.
+
+## 3.2. Access Control
+
+How access control is defined in a Credential Service is implementation-specific. Implementations may
+provide the ability to selectively restrict access to resources. The access control mechanism must support the scope
+restrictions defined in [Section 3.1](#31-access-scopes). Implementations may support additional restriction methods,
+including requiring the requester to present its own VPs.
+
+# 4. Resolution API
+
+If a client is not authorized for an endpoint request, the Credential Service may
+return `401 Not Authorized`, `403 Forbidden`, or `404 Not Found`.
```

Comment:
404 should be reserved for when a resource is not present. it's misleading to return 404 after an auth problem.